src/main/java/modtrekt/logic/parser/AddCommandParser.java

```diff
@@ -39,10 +39,12 @@ public Command parse(String args) throws ParseException {
                 ArgumentTokenizer.tokenize(args, CliSyntax.PREFIX_MOD_NAME, CliSyntax.PREFIX_MOD_CODE,
                         CliSyntax.PREFIX_MOD_CREDIT, CliSyntax.PREFIX_TASK);
 
-        if (arePrefixesPresent(argMultimap, CliSyntax.PREFIX_TASK)) {
+        if (arePrefixesPresent(argMultimap, CliSyntax.PREFIX_TASK, CliSyntax.PREFIX_MOD_CODE)) {
```

Comment:
Can `arePrefixesPresent` be refactored out of each `CommandParser` class and into `ParserUtils` or `ArgumentTokenizer`?

There seems to be lot of code duplication involving that method.

Reply:
Great point - just refactored it out in the latest commit. Should be good to merge?

Reply:
Good job refactoring and resolving the conflicts!